R/archive.R

```diff
@@ -104,6 +119,10 @@ epi_archive =
 #'   fields; named entries from the passed list or will be included as well.
 #' @return An `epi_archive` object.
 #' @importFrom data.table as.data.table key setkeyv
+#' 
+#' @details 
+#' Refer to the documentation for `as_epi_archive()` for more information 
```

Comment:
Minor improvement but I found online that we can actually link the function help page in roxygen using `\code{\link{as_epi_archive}}`. Might be helpful here?

https://stackoverflow.com/questions/25489042/linking-to-other-packages-in-documentation-in-roxygen2-in-r

Reply:
Thanks, Chloe! Based on a quick look into this, I think that one of the easiest ways to link to another function in the package we're working on is to use something like `[as_epi_archive()]` in the code, which translates to `\code{\link[=as_epi_archive]{as_epi_archive()}}` in the Rd file. I've now done that where I stated to look to the `as_epi_archive()` function in the `epi_archive` documentation.